compute/compute-node.Dockerfile

```diff
@@ -1654,9 +1663,11 @@ RUN echo -e "--retry-connrefused\n--connect-timeout 15\n--retry 5\n--max-time 30
 # Layer "awscli"
 #
 #########################################################################################
-FROM alpine/curl:${ALPINE_CURL_VERSION} AS awscli
+FROM build-deps AS awscli
 ARG TARGETARCH
 RUN set -ex; \
+    apt update; \
+    apt install --no-install-recommends -y unzip; \
```

Comment:
Can you please clean the cache after apt update as it's done for other steps? https://github.com/neondatabase/neon/blob/d0ca1d5d5bba1c6ecb37418815e1724fc2f54054/compute/compute-node.Dockerfile#L142

Reply:
Didn't spot that, thanks!